.github/workflows/deploy.yml

```diff
@@ -270,7 +270,7 @@ jobs:
           channel-id: ${{ secrets.SLACK_RELEASE_CHANNEL_ID }}
           payload: |
             {
-              "text": "*Release: <${{ env.CDN_URL }}|${{ env.CURRENT_VERSION_VALUE }}>*\n${{ env.DATE }}\nCC: <!subteam^S0555JBV36D>",
+              "text": "*Release: <${{ env.CDN_URL }}|${{ env.CURRENT_VERSION_V1_VALUE }}>*\n${{ env.DATE }}\nCC: <!subteam^S0555JBV36D>",
```

Comment:
What is the value of CURRENT_VERSION_V1_VALUE?

Reply:
It's the version of the SDK v1.1 package. Refer to Ln 73 in the same file.